DQM/Physics/src/TopSingleLeptonDQM.cc

```diff
@@ -415,38 +433,53 @@ void MonitorEnsemble::fill(const edm::Event& event,
     reco::GsfElectronRef gsf_el = elec->gsfElectronRef();
     // restrict to electrons with good electronId
     if (electronId_.isUninitialized() ? true : ((double)(*electronId)[gsf_el] >=
-                                                eidCutValue_)) {
+                                                eidCutValue_)) { //This Electron Id is not currently used, but we can keep this for future needs
       if (!elecSelect_ || (*elecSelect_)(*elec)) {
 
         double el_ChHadIso = gsf_el->pfIsolationVariables().sumChargedHadronPt;
         double el_NeHadIso = gsf_el->pfIsolationVariables().sumNeutralHadronEt;
         double el_PhIso = gsf_el->pfIsolationVariables().sumPhotonEt;
-        double el_pfRelIso =
-            (el_ChHadIso +
-             max(0., el_NeHadIso + el_PhIso -
-                         0.5 * gsf_el->pfIsolationVariables().sumPUPt)) /
-            gsf_el->pt();
-        if (eMult == 0) {
-          // restrict to the leading electron
-          fill("elecPt_", elec->pt());
-          fill("elecEta_", elec->eta());
+				double absEta = abs(gsf_el->superCluster()->eta());
```

Comment:
here and elsewhere, use std::abs (likely its a bug here to make an integer based comparison)

Reply:
changed to 
double absEta = std::fabs(gsf_el->superCluster()->eta());